code/datums/mutations.dm

```diff
@@ -147,6 +168,10 @@ var/thanks_tobba = 'icons/fonts/runescape_uf.ttf'
 	if(..())
 		return
 	owner.status_flags |= CANSTUN | CANWEAKEN | CANPARALYSE | CANPUSH
+	owner.client.prefs.features["mcolor"] = naturalcolor
```

Comment:
for the record i didn't even write that, I just added new sprites to it don't hit me